Makefile

```diff
@@ -17,7 +17,7 @@ COMMIT_HASH ?= $(shell git rev-parse HEAD)
 # have been built before trying to create a kiali server container
 # image. The UI project is configured to place its build
 # output in the $UI_SRC_ROOT/build/ subdirectory.
-CONSOLE_LOCAL_DIR ?= ${ROOTDIR}/../../../../../kiali-ui
+CONSOLE_LOCAL_DIR ?= ${ROOTDIR}/../kiali-ui
```

Comment:
This is already in master (see https://github.com/kiali/kiali/blob/master/Makefile#L20) so I suspect this PR needs to be rebased on the latest master.

Reply:
I'm also not convinced we want this PR yet. More discussions are probably needed around multi-cluster support.

Reply:
Sure I will close this PR, and send a more complete feature PR in multi cluster area after some careful discussions:)